core/services/ocr2/plugins/directrequestocr/scripts/gen_oracle_contract_config.go

```diff
@@ -14,6 +16,14 @@ import (
 	"github.com/smartcontractkit/chainlink/core/services/ocr2/plugins/directrequestocr/config"
 )
 
+type OCR2DROracleConfig struct {
```

Comment:
All names (type + all fields) can be lowercase, which means package-level visibility instead of public

Reply:
Apparently they need to be uppercase for JSON marshal
"Struct fields must start with upper case letter (exported) for the JSON package to see their value."

Fixed the struct name though.